packages/web/resolvers/comment.ts

```diff
@@ -121,26 +121,39 @@ const createComment = async ({
     },
   })
 
-  await Promise.all(thread.subscriptions.map(async ({ user }: { user: User }) => {
-    if (user.id === author.id) {
-      // This is the user creating the comment, do not notify them.
-      return new Promise((res) => res(null))
-    }
-
-    await createEmailNotification(db, user, {
-      type: EmailNotificationType.THREAD_COMMENT,
-      comment,
-    })
+  const updatedThreadData = await db.thread.findUnique({
```

Comment:
You can save a join here by just selecting the subscriptions directly, e.g.:

```tsx
const subscriptions = db.subscriptions.find({
  where: { threadId: thread.id },
  include: { user: true },
})
```

Which should also fix the issue below about the missing `user` key when destructuring in the  map  callback